popup_windows.js

```diff
@@ -0,0 +1,133 @@
+/* eslint-disable no-multi-str */
+/* eslint-disable no-plusplus */
+/* eslint-disable quotes */
+
+const container = document.createElement('div');
```

Comment:
- [ ] Modal popup data is populated with the data from the projects array. (Currently, it is being populated by appending HTML code to the container, please fix this)